ui/lib/sync/addon/components/sync-header.hbs

```diff
@@ -17,6 +17,9 @@
       {{#if this.version.isCommunity}}
         <Hds::Badge @text="Enterprise feature" @color="highlight" @size="large" />
       {{/if}}
+      {{#if this.version.isEnterprise}}
```

Comment:
I made these separate conditionals instead of an `{{else}}` situation to make beta removal easier